mycore-base/src/main/java/org/mycore/common/config/MCRProperty.java

```diff
@@ -0,0 +1,38 @@
+package org.mycore.common.config;
```

Comment:
it might be better to put annotations into their own subpackage:
org.mycore.common.config.annotation